src/Gtin/Factory.php

```diff
@@ -8,21 +8,24 @@
 final class Factory
 {
     /**
-     * @throws NonNormalizable
+     * @param string $value
+     * @param array $customPrefixes
+     *
+     * @return Gtin
      */
-    public static function create(string $value): Gtin
+    public static function create(string $value, array $customPrefixes = []): Gtin
```

Comment:
Does it make sense to rewrite this mechanizm the following way:
- Factory::create remains as it was before;
- Constructor of `Real\Validator\Gtin\General` does not throw an expection;
- Instead `Real\Validator\Gtin\General` exposes new public method "validate", which accepts $customPrefixes as parameter. Validate method will actually verify value and throw exception in case of failure.